crates/dprint/src/utils/glob_utils.rs

```diff
@@ -61,6 +61,15 @@ pub fn to_absolute_glob(pattern: &str, dir: &str) -> String {
   let mut pattern = pattern.replace("\\", "/");
   let dir = dir.replace("\\", "/");
 
+  // .gitignore: "If there is a separator at the beginning or middle (or both) of
+  // the pattern, then the pattern is relative to the directory level of the particular
+  // .gitignore file itself. Otherwise the pattern may also match at any level below the
+  // .gitignore level."
+  let is_relative = match pattern.find("/") {
```

Comment:
negated globs follow this as well, no (this is a real question, I don't know the exact answer, my hunch is yes)? So a preceding `!` should be stripped if there is any?